Support python3

I turned off python3 checking while implementing tests in #4

The argument count to `CodeType` changes in python3, here's the current failure mode:

```
_________________________________ test_verbose _________________________________

venv = ns(before=local('/tmp/pytest-of-asottile/pytest-47/test_verbose0/before'), after=local('/tmp/pytest-of-asottile/pytest-47/test_verbose0/after'))
capsys = <_pytest.capture.CaptureFixture object at 0x7f22723eb278>

    def test_verbose(venv, capsys):
>       run(venv.before, venv.after, args=['--verbose'])

tests/virtualenv_tools_test.py:84: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/virtualenv_tools_test.py:23: in run
    [before.strpath, '--update-path={}'.format(after.strpath)] + args,
virtualenv_tools.py:309: in main
    if not update_paths(path, options.update_path):
virtualenv_tools.py:211: in update_paths
    update_pycs(lib_dir, new_path, lib_name)
virtualenv_tools.py:159: in update_pycs
    update_pyc(filename, local_path)
virtualenv_tools.py:137: in update_pyc
    new_code = _process(code)
virtualenv_tools.py:134: in _process
    code = _make_code(code, new_path, consts)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

code = <code object <module> at 0x7f22723eff60, file "/tmp/pip-build-1cwhvgdb/setuptools/easy_install.py", line 1>
filename = '/tmp/pytest-of-asottile/pytest-47/test_verbose0/after/site-packages/__pycache__/easy_install.cpython-35.pyc'
consts = ['Run the EasyInstall command', '__main__', 0, ('main',), None]

    def _make_code(code, filename, consts):
        return CodeType(code.co_argcount, code.co_nlocals, code.co_stacksize,
                        code.co_flags, code.co_code, tuple(consts),
                        code.co_names, code.co_varnames, filename,
                        code.co_name, code.co_firstlineno, code.co_lnotab,
>                       code.co_freevars, code.co_cellvars)
E       TypeError: an integer is required (got type bytes)

virtualenv_tools.py:125: TypeError
---------------------------- Captured stdout setup -----------------------------
Using real prefix '/usr'
Path not in prefix '/home/asottile/workspace/virtualenv-tools/.tox/py35/include/python3.5m' '/usr'
New python executable in /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/python3.5
Also creating executable in /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/python
Installing setuptools, pip, wheel...done.
----------------------------- Captured stdout call -----------------------------
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/pip3
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/easy_install
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/python-config
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/easy_install-3.5
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/pip3.5
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/pip
S /tmp/pytest-of-asottile/pytest-47/test_verbose0/before/bin/wheel
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,7 @@ language: python
 python: 3.5
 env: # These should match the tox env list
     - TOXENV=py27
+    - TOXENV=py35
 install: pip install coveralls tox
 script: tox
 after_success: coveralls

--- a/tox.ini
+++ b/tox.ini
@@ -1,6 +1,6 @@
 [tox]
 project = pre_commit
-envlist = py27
+envlist = py27,py35
 
 [testenv]
 deps = -rrequirements-dev.txt
@@ -8,7 +8,7 @@ commands =
     coverage erase
     coverage run -m pytest {posargs:tests}
     # TODO: change to 100
-    coverage report --show-missing --fail-under 75
+    coverage report --show-missing --fail-under 74
     pre-commit install -f --install-hooks
     pre-commit run --all-files
 

--- a/virtualenv_tools.py
+++ b/virtualenv_tools.py
@@ -118,11 +118,22 @@ def update_pyc(filename, new_path):
             raise
 
     def _make_code(code, filename, consts):
-        return CodeType(code.co_argcount, code.co_nlocals, code.co_stacksize,
-                        code.co_flags, code.co_code, tuple(consts),
-                        code.co_names, code.co_varnames, filename,
-                        code.co_name, code.co_firstlineno, code.co_lnotab,
-                        code.co_freevars, code.co_cellvars)
+        if sys.version_info[0] == 2:  # pragma: no cover (PY2)
+            arglist = [
+                code.co_argcount, code.co_nlocals, code.co_stacksize,
+                code.co_flags, code.co_code, tuple(consts), code.co_names,
+                code.co_varnames, filename, code.co_name, code.co_firstlineno,
+                code.co_lnotab, code.co_freevars, code.co_cellvars,
+            ]
+        else:  # pragma: no cover (PY3)
+            arglist = [
+                code.co_argcount, code.co_kwonlyargcount, code.co_nlocals,
+                code.co_stacksize, code.co_flags, code.co_code, tuple(consts),
+                code.co_names, code.co_varnames, filename, code.co_name,
+                code.co_firstlineno, code.co_lnotab, code.co_freevars,
+                code.co_cellvars,
+            ]
+        return CodeType(*arglist)
 
     def _process(code):
         consts = []

